src/hosted.html

```diff
@@ -21,7 +21,14 @@
     </style>
 </head>
 <body>
+<script src="https://togetherjs.com/togetherjs-min.js"></script>
+<button onclick="TogetherJS(this); return false;">Start TogetherJS</button>
     <div id="bramble" class="full"></div>
     <script src="thirdparty/require.min.js" data-main="hosted.js"></script>
-</body>
+    <script>
+        TogetherJS.hub.on('data', function (delta) {
+            console.log("bhai ka aaya data" + delta);
```

Comment:
Yup sorry about that. Completely missed that out. 
It means "Here's your data Brother". 

Will get rid of it.